### ISSUE TITLE
loss function错了

### ISSUE THREAD

**Issue by h3z** [h3z/yolo1-tensorflow2#1]
理解错了，一开始是在吴恩达dl课程里知道的yolo，然后习惯了他讲的一个多个anchor可以负责检测格子里多个目标。
导致看论文时候也就这么理解了。
直到实现loss function的时候，发现怎么都解释不通😞

回去又看了吴恩达讲的yolo，才发现他的输出结构是，cell * cell * anchor * (5 + class)，而论文中是 cell * cell * (anchor * 5 + class)

然后这里讲的就很清楚
https://medium.com/@jonathan_hui/real-time-object-detection-with-yolo-yolov2-28b1b93e2088

然后这是课程里提到的
<img width="734" alt="image" src="https://user-images.githubusercontent.com/28476985/80286869-8e2ac800-8760-11ea-9087-570b79100977.png">
